frontend/src/metabase/public/components/widgets/DisplayOptionsPane.jsx

```diff
@@ -30,63 +34,88 @@ const DisplayOptionsPane = ({
+            onChange={e =>
+              onChangeDisplayOptions({
+                ...displayOptions,
+                bordered: e.target.checked,
+              })
+            }
+          />
+          <CheckBox
+            label={t`Title`}
+            checked={displayOptions.titled}
+            onChange={e =>
+              onChangeDisplayOptions({
+                ...displayOptions,
+                titled: e.target.checked,
+              })
+            }
+          />
+        </StyleContainer>
+      </DisplayOptionSection>
+      <DisplayOptionSection title={t`Appearance`}>
+        <Radio
+          value={displayOptions.theme}
+          options={THEME_OPTIONS}
+          onChange={value =>
+            onChangeDisplayOptions({ ...displayOptions, theme: value })
           }
-        />
-      </StyleContainer>
-    </DisplayOptionSection>
-    <DisplayOptionSection title={t`Appearance`}>
-      <Radio
-        value={displayOptions.theme}
-        options={THEME_OPTIONS}
-        onChange={value =>
-          onChangeDisplayOptions({ ...displayOptions, theme: value })
-        }
-        variant="normal"
-        showButtons
-        vertical
-      />
-    </DisplayOptionSection>
-    {canWhitelabel && (
-      <DisplayOptionSection title={t`Font`}>
-        <Select
-          value={displayOptions.font}
-          options={MetabaseSettings.get("available-fonts").map(font => ({
-            name: font,
-            value: font,
-          }))}
-          onChange={e => {
-            onChangeDisplayOptions({
-              ...displayOptions,
-              font: e.target.value,
-            });
-          }}
+          variant="normal"
+          showButtons
+          vertical
         />
       </DisplayOptionSection>
-    )}
-  </div>
-);
+      {canWhitelabel && (
+        <>
+          <DisplayOptionSection title={t`Font`}>
+            <Select
+              value={displayOptions.font}
+              options={MetabaseSettings.get("available-fonts").map(font => ({
+                name: font,
+                value: font,
+              }))}
+              onChange={e => {
+                onChangeDisplayOptions({
+                  ...displayOptions,
+                  font: e.target.value,
+                });
+              }}
+            />
+          </DisplayOptionSection>
+          <DisplayOptionSection title={t`Download data`}>
+            <ToggleContainer>
+              <ToggleLabel
+                htmlFor={toggleId}
+              >{t`Enable users to download data from this embed?`}</ToggleLabel>
+              <Toggle
+                id={toggleId}
+                aria-checked={!displayOptions.hide_download_button}
+                role="switch"
+                value={!displayOptions.hide_download_button}
+                onChange={isEnabled => {
+                  onChangeDisplayOptions({
+                    ...displayOptions,
+                    hide_download_button: !isEnabled ? true : null,
```

Comment:
@ranquild I flipped the flag, and also remove the parameter to make the embed code cleaner. e.g.
#### default: show download button
![image](https://user-images.githubusercontent.com/1937582/175036184-9a5f1d5d-5c12-4ded-95ae-96c79e0b41fe.png)

#### hide download button
![image](https://user-images.githubusercontent.com/1937582/175036237-8f94ebbd-8dec-47ae-8eef-a1fa17cfbbb0.png)